nuxeo-platform-user-registration-web/src/main/resources/OSGI-INF/l10n/messages_fr_FR.properties

```diff
@@ -16,6 +16,6 @@ label.request.date.range=Dates
 label.request.help.range=Dates de validit\u00E9 des droits sur le document.
 label.request.groups=Groups
 label.unicity.usernamepwd=Cet utilisateur existant dans Nuxeo dispose d'un autre email.
-label.unicity.sameemail=L'utilisateur {0}, existant dans Nuxeo dispose du m\u00EAme email.
+label.unicity.sameemail=L''utilisateur {0}, existant dans Nuxeo dispose du m\u00EAme email.
 label.error.requestNotExisting=Il n'existe aucune demande d'enregistrement avec l'identifiant {0}
```

Comment:
I believe it is the same for next line i.e. `Il n''existe aucune demande d''enregistrement avec l''idetifiant {0}`